### ISSUE TITLE
No audio available on dev server

### ISSUE THREAD

**Issue by mramosch** [Tatoeba/tatoeba2#3003]
Sentences that have audio don’t show an availability speaker icon.

A search for sentences with ‘audio=yes’ returns:
• An Internal Error Has Occurred

**Comment by ckjpn** [comment]
I didn't get an internal error message for this search.

https://dev.tatoeba.org/en/sentences/search?from=eng&has_audio=yes&native=yes&orphans=no&query=driver&sort=random&trans_filter=limit&unapproved=no

However, the speaker icon isn't the "play" icon, but the "audio not available" one.

**Comment by mramosch** [comment]
No search word
Lang to: English
Audio: yes

**Comment by ckjpn** [comment]
I don't get the same results every time.
Sometimes, I can see the "play" button, sometimes not.

Sorted on shortest first, I didn't get an internal error message, even without a query word.

(Perhaps that helps identify the problem.)

https://dev.tatoeba.org/en/sentences/search?from=eng&has_audio=yes&native=yes&orphans=no&query=&sort=words&sort_reverse=&tags=&to=&trans_filter=limit&trans_has_audio=&trans_link=&trans_orphan=&trans_to=&trans_unapproved=&trans_user=&unapproved=no&user=

I didn't experiment with other sorts.

**Comment by mramosch** [comment]
There is a new Play md-button wrapper in the code, which includes the symbol assignment etc.

Maybe something went wrong there… ?

**Comment by trang** [comment]
The search is a bit buggy on the dev server and we never took the time, nor had the time, to really make it work consistently.

@mramosch Is this blocking you from doing something?

**Comment by jiru** [comment]
This bug is likely a result of #2880.

In the past, there used to be no actual audio on dev.tatoeba.org, it just blindly linked to files on tatoeba.org, which worked 99℅ of the time (it could fail if the sentence had been removed or something).

In order to have a proper test environment for audios on dev.tatoeba.org, it now has its own, separate set of audio files, which only consists of a few files I used to test out multiple files per sentences. I think we just don't have enough disk space on dev to store many audios.

Probably the search index just needs to be refreshed for all languages that used to have audio on dev. As for the "An Internal Error Has Occurred", I suppose it's a different issue.

**Comment by mramosch** [comment]
@trang 
Not really, but when I am developing I assume that from a functional point of view the behavior of the dev server is identical to the production server. So I might be looking for bugs in my code…

I also experience the same behavior on sentence pages where the speaker symbol sometimes shows up and sometimes appears grayed out on the same sentence…
